### PR TITLE
contrib/hugo: new package (0.124.1)

### PR DESCRIPTION
**Pull request by ifreund**

### CODE REVIEW

File: contrib/hugo/template.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+build_style = "go"
+hostmakedepends = ["go"]
+pkgdesc = "Fast and flexible static site generator"
+maintainer = "Isaac Freund <mail@isaacfreund.com>"
+license = "Apache-2.0"
+url = "https://gohugo.io"
+source = f"https://github.com/gohugoio/hugo/archive/v{pkgver}.tar.gz"
+sha256 = "0beb0436f6bd90abb425523229a37f1d31e2e9c7ba9fac4556a72aab3b11bfef"
+# tests require network access
+options = ["!check"]
```
Comment (mhmdanas) [inline contrib/hugo/template.py:13]:
```suggestion
# tests require network access
# manpages and completions are generated by running binary so no cross
options = ["!check", "!cross"]
```

**Review by mhmdanas — CHANGES_REQUESTED** [review]

File: contrib/hugo/template.py (hunk 1 above)
Comment (ifreund) [reply contrib/hugo/template.py]: Added, thanks

File: contrib/hugo/template.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+pkgname = "hugo"
+pkgver = "0.124.1"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+pkgdesc = "Fast and flexible static site generator"
```
Comment (nekopsykose) [inline contrib/hugo/template.py:6]:
```suggestion
pkgdesc = "Static site generator"
```

Comment (ifreund) [reply contrib/hugo/template.py]: done :)

**Pull request merged**